README.md

```diff
@@ -1,13 +1,4 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
+This repo contains Bogdan Ciobanu's portfolio and STEP projects.
 
```

Comment:
I suggest you include a bit more information, such as an overview of the various pieces and where are they located.
Think about the questions someone who just landed on this project unfamiliar with the project: what's this portfolio all about? How do I set this project up so I can see it working? How do I run tests, etc.? I suggest including a simple "getting started" section with commands guiding the user through the setup process, with end goal of: here, the webpage is loaded and I see the portfolio.